oojs.md

```diff
@@ -24,13 +34,19 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+class Album extends Playlist={
+
+  artist:function(artist){
+    this.artist=artist;
+  }
+  Album.push(artist)
+}
 ```
 
 ### 4
 
 Create an instance of the `Album` class.
 
 ```js
-Your answer goes here
+var album1=new Album("Mymusic")
```

Comment:
You also need to pass in the parameters for the Playlist class. Remember, `Album` inherits from `Playlist`, meaning we need to think about the latter's parameters.